src/clib/pio_getput_int.c

```diff
@@ -1535,12 +1535,35 @@ int PIOc_put_vars_tc(int ncid, int varid, const PIO_Offset *start, const PIO_Off
                 /* This is a scalar var. */
                 LOG((2, "pnetcdf writing scalar with ncmpi_put_vars_*() file->fh = %d varid = %d",
                      file->fh, varid));
-                pioassert(!start && !count && !stride, "expected NULLs", __FILE__, __LINE__);
```

Comment:
Do we also need to remove a similar assert on line 1254 for PIO_IOTYPE_ADIOS?

Reply:
Ok, I will replace it with warnings too